scripts/build_linux.sh

```diff
@@ -5,17 +5,17 @@ aws ecr get-login-password --region us-west-1 | docker login --username AWS --pa
 go mod vendor
 
 # Build linux/amd64
-docker build . --file ./docker/Dockerfile_linux_amd64 --tag cli-linux-amd64-builder-image --build-arg SECRET_KEY="$SECRET_KEY" --build-arg SECRET_PASSPHRASE="$SECRET_PASSPHRASE" # not finalized
+docker build . --file ./docker/Dockerfile_linux_amd64 --tag cli-linux-amd64-builder-image --build-arg NFPM_PASSPHRASE="$NFPM_PASSPHRASE" # not finalized
```

Comment:
FYI, this file will be replaced with the following make target: https://github.com/confluentinc/cli-release/pull/91/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R135-R147

Reply:
That makes is easier, probably. Ultimately the point of this line is to make the GPG passphrase available to goreleaser, since it pulls it from a specific environment variable.

All we need to do in the cli-release target is set that env var.